packages/valory/skills/market_creation_manager_abci/rounds.py

```diff
@@ -746,7 +763,11 @@ class MarketCreationManagerAbciApp(AbciApp[Event]):
     """MarketCreationManagerAbciApp"""
 
     initial_round_cls: AppState = CollectRandomnessRound
-    initial_states: Set[AppState] = {CollectRandomnessRound, PostTransactionRound}
+    initial_states: Set[AppState] = {
+        CollectRandomnessRound,
+        PostTransactionRound,
+        CloseMarketsRound,
```

Comment:
Why?

Reply:
the way it was previously wired, CloseMarkets can end up not being called. Now it is the first behaviour that is called after registration, in order to do that it must be a initial state.